Library/Homebrew/test/formula_installer_bottle_spec.rb

```diff
@@ -72,6 +73,40 @@ def temporarily_install_bottle(formula)
     end
   end
 
+  specify "basic bottle install with cellar information on sha256 line" do
+    allow(DevelopmentTools).to receive(:installed?).and_return(false)
+    Homebrew.install_args.parse(["testball_bottle_cellar"])
+    temporarily_install_bottle(TestballBottle.new) do |f|
+      # Copied directly from formula_installer_spec.rb
+      # as we expect the same behavior.
+
```

Comment:
```suggestion
```

I don't think where it's copied from is relevant. If you need to have identical behaviour: I suggest sharing logic between the tests.

Reply:
I removed the comment but still need to share the logic between the tests.